modules/ding_event/ding_event.module

```diff
@@ -344,6 +357,24 @@ function ding_event_form_views_exposed_form_alter(&$form, &$form_state) {
     if (empty($input)) {
       $form['reset']['#access'] = FALSE;
     }
+
+    if (!empty($form['#info']['filter-term_node_tid_depth'])) {
+      $label = $form['#info']['filter-term_node_tid_depth']['label'];
+      unset($form['#info']['filter-term_node_tid_depth']['label']);
+      $form['event_category_term_node_tid_depth']['#title'] = '<h2 class="list-title sub-menu-title">' . $label . '</h2>';
+    }
```

Comment:
https://inlead.atlassian.net/browse/EASYOPAC-909

Reply:
This seems legit as well to get into main distro. However I don't understand why `field_ding_event_category_tid` is renamed to `term_node_tid_depth`.

Reply:
Main distro uses the new key - https://github.com/easySuite/ding2/blob/development/modules/ding_event/ding_event.views_default.inc#L329

Reply:
So if this was already merged, what is then the issue? https://github.com/easySuite/ding2/pull/468/files